_posts/2020-04-01-cockpit-216.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 216
+author: lis
+date: 2020-04-01
+tags: cockpit linux
+slug: cockpit-216
+category: release
+summary: Cockpit with improved SELinux page and last login banner
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 216.
+
+### SELinux: Automatic application of solutions
+
+The SELinux page now has "Apply this solution" buttons for suggested
+solutions to logged access control errors.
+
+![Apply this solution](/images/selinux-apply-this-solution.png)
+
+The System Modifications table will be updated after applying a
+solution.
+
+### Machines: Drop virsh backend support
+
+This release drops the virsh backend support from the machines page and
+switches over to using libvirt-dbus exclusively.  As a result, as of this
+release, cockpit-machines is no longer supported on Ubuntu 18.04.
+
+### New "last login" banner
```

Comment:
I'd prefix with "Overview:" to keep the "Page: What" pattern. I'd  also drop "New", as everything in release notes is supposed to be new. :-)